### PR TITLE
ENH: Interactive `tabulate` viz

### LINKED ISSUE #1: `tabulate` viz
Migrated from https://github.com/qiime2/qiime2/issues/271

### PR DESCRIPTION
**Pull request by thermokarst**
Renders an HTML/JS table of supplied Metadata. Allows users to sort,
filter, and export the tabular Metadata.

[Sample Viz](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fforum-qiime2-org.s3-us-west-2.amazonaws.com%2Foriginal%2F1X%2F078d253d6d6bb78ff685fba7f3edfae90a518533.qzv)

Fixes #1

### CODE REVIEW

**Comment by gregcaporaso** [comment]
@thermokarst, this looks great! @jairideout and I reviewed and only have one request - is it possible to have all values quoted in the `Copy` export so that the copied text looks the same as the text in the `TSV` export?

**Comment by thermokarst** [comment]
Thanks @gregcaporaso, good catch! I think I got that [under control](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fforum-qiime2-org.s3-us-west-2.amazonaws.com%2Foriginal%2F1X%2Fe2a56207de38d600f8d9b01ad105f18f3115fc16.qzv). RfR.

File: q2_metadata/plugin_setup.py [hunk 1]
```diff
@@ -22,3 +24,21 @@
...
+    parameters={
+        'input': qiime2.plugin.Metadata,
+        'page_size': qiime2.plugin.Int,
+    },
+    parameter_descriptions={
+        'input': 'The metadata to tabulate.',
+        'page_size': 'The maximum number of Metadata records to display '
+                     'per page',
+    },
+    name='tabulate',
```
Comment (thermokarst) [inline q2_metadata/plugin_setup.py]: **doh**

Comment (jairideout) [inline q2_metadata/plugin_setup.py]: Looking at existing `names`, there doesn't seem to be a language/style convention... so whatever you think sounds and looks good :+1:

**Comment by thermokarst** [comment]
Thanks @jairideout! I think I have addressed your comments!

**Pull request merged**